hoot-core/src/main/cpp/hoot/core/info/ApiEntityDisplayer.cpp

```diff
@@ -184,26 +170,49 @@ void printApiEntities2(const std::string& apiEntityClassName)
         QString line = "  " + name + QString(indentAfterName, ' ');
         line += description.description;
         if (description.experimental)
-        {
           line += " (experimental)";
-        }
         LOG_VARD(line);
         output.append(line);
       }
     }
   }
-
   output.sort();
+
+  QString buffer;
+  QTextStream ts(&buffer);
   for (int i = 0; i < output.size(); i++)
+    ts << output.at(i) << endl;
+
+  return ts.readAll();
+}
+
+QString ApiEntityDisplayer::_apiEntityTypeForBaseClass(const QString className)
+{
+  LOG_VARD(className);
+  if (className.toStdString() == OsmMapOperation::className() ||
+      Factory::getInstance().hasBase<OsmMapOperation>(className.toStdString()))
+  {
+    return "operation";
+  }
+  else if (className.toStdString() == ElementVisitor::className() ||
+           Factory::getInstance().hasBase<ElementVisitor>(className.toStdString()))
   {
-    std::cout << output.at(i) << std::endl;
+    return "visitor";
   }
+  else if (className.toStdString() == ConstElementVisitor::className() ||
+           Factory::getInstance().hasBase<ConstElementVisitor>(className.toStdString()))
+  {
+    return "visitor (const)";
+  }
+  return "";
 }
 
-void ApiEntityDisplayer::displayCleaningOps()
+QString ApiEntityDisplayer::displayCleaningOps()
 {
```

Comment:
Should probably be something like getCleaningOpsStr now or similar.

Reply:
Good point.  I did it with other classes but missed this one.  I'll make the change.